src/message_interface/response.rs

```diff
@@ -49,7 +49,9 @@ pub enum Response {
     /// [`BuildBasicOutput`](crate::message_interface::AccountMethod::BuildBasicOutput)
     /// [`BuildFoundryOutput`](crate::message_interface::AccountMethod::BuildFoundryOutput)
     /// [`BuildNftOutput`](crate::message_interface::AccountMethod::BuildNftOutput)
-    BuiltOutput(OutputDto),
+    /// [`GetFoundryOutput`](crate::message_interface::AccountMethod::GetFoundryOutput)
+    /// [`PrepareOutput`](crate::message_interface::AccountMethod::PrepareOutput)
+    OutputDto(OutputDto),
```

Comment:
For consistency, I am not sure we should have `Dto` in the name, see `Account(AccountDto)` for example. I understand `Output` is already taken by `Output(Option<Box<OutputDataDto>>)` but maybe this one should be `OutputData(Option<Box<OutputDataDto>>)` instead? What do you think @Thoralf-M ?

Reply:
I agree as well, I will implement it. Sidenote: do you think `Outputs(Vec<OutputDataDto>)` should also be renamed then? It's a bit tricky since `OutputData` is already plural.

Reply:
Hmm yeah that's tricky. `OutputsData` or `OutputsDatas` ? Idk 😬

Reply:
I think `OutputsData` is okay. I can't think of anything better at least 😄